public/pages/AnomalyCharts/components/FeatureChart/NoFeaturePrompt.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from 'react';
+
+import {
+  EuiEmptyPrompt,
+  EuiText,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiButton,
+} from '@elastic/eui';
+
+import ContentPanel from '../../../../components/ContentPanel/ContentPanel';
+
+interface NoFeaturePromptProps {
+  detectorId: string;
+  title?: string;
```

Comment:
Is there any case where `title` is used?

Reply:
Good catch, `title` not used currently, will remove this